src/main/java/uk/gov/hmcts/reform/userprofileapi/util/UserProfileValidator.java

```diff
@@ -130,4 +129,17 @@ static void validateUserIds(GetUserProfilesRequest getUserProfilesRequest) {
             throw new RequiredFieldMissingException("no user id in request");
         }
     }
+
+    static void validateUserProfileDataAndUserId(UpdateUserProfileData userProfileData, String userId) {
+
+        if (null == userProfileData) {
+
+            throw new RequiredFieldMissingException("No Request Body in the request");
+        } else if (StringUtils.isBlank(userId)
+                || (!CollectionUtils.isEmpty(userProfileData.getRolesAdd()) && userProfileData.getRolesAdd().size() == 0)

```

Comment:
CollectionUtils.isEmpty internally checks null and size both, so I doubt size checking is required

Reply:
it checks only empty not the size, tests fail when we don't include the size check

Reply:
ok. fine.